app/javascript/styles/mastodon/components.scss

```diff
@@ -7969,7 +7969,7 @@ noscript {
   width: 600px;
   background: $ui-base-color;
   border-radius: 8px;
-  overflow: hidden;
+  overflow: auto;
```

Comment:
I think it'd be best to have `overflow-x: hidden` and `overflow-y: auto` to avoid long display names or long server names messing with the layout.